### DISCUSSION TITLE
Custom Liquid Filter in template intellisence

### DISCUSSION THREAD

**Discussion by craftyweb** [OrchardCMS/OrchardCore#17086] [General]
Hello,

Just wondering if it's possible to add your custom created LiquidFilter in the liquid intellisence when you're creating a template.

I found the ScriptsMiddleware in the OrchardCore.Liquid module but It's never triggered. 

It would be nice if I can show my custom filters there as well.

Thanks!

**Comment by Piedone** [comment]
The editor there is Monaco, what you can extend, see: https://microsoft.github.io/monaco-editor/typedoc/interfaces/languages.CompletionItemProvider.html And related: https://github.com/OrchardCMS/OrchardCore/issues/2141
